src/d3a/setup/sam_config_const_load.py

```diff
@@ -38,8 +38,7 @@ def get_setup(config):
                          appliance=PVAppliance()),
                     Area('H1 Storage1',
                          strategy=StorageStrategy(battery_capacity_kWh=12 * 1.2,
-                                                  max_abs_battery_power_kW=12 * 1.2,
-                                                  initial_capacity_kWh=0.6 * 12 * 1.2),
+                                                  max_abs_battery_power_kW=12 * 1.2),
```

Comment:
[It](https://github.com/gridsingularity/d3a/pull/519#discussion_r330047979) needs to adapted here as well.